manticore/core/manticore.py

```diff
@@ -1219,7 +1223,28 @@ def save_run_data(self):
         with self._output.save_stream("manticore.yml") as f:
             config.save(f)
 
+        with self._output.save_stream("global.solver_stats") as f:
+            for s, n in sorted(SOLVER_STATS.items()):
+                f.write("%s: %d\n" % (s, n))
+
+        if SOLVER_STATS["timeout"] > 0 or SOLVER_STATS["unknown"] > 0:
+            logger.info(
+                "Warning: the SMT solvers returned timeout or unknown for certain program paths. Results could not cover the entire set of possible paths"
```

Comment:
* Since the log level for this is `info`, we should remove the "Warning" at the beginning of the message.
* How often do timeouts occur? It looks like the default log level for this file is `info`, so this message will be printed after any run in which a solver times. Personally I think that might be a little too verbose, but we can discuss further.

Reply:
The log level for this was changed to `warning`. Timeouts can happen in any non-trivial smart contract code (specially with the default timeout of two minutes), however this message will be printed once per execution, at the very end, so it should not so be verbose.